commcare_connect/opportunity/visit_import.py

```diff
@@ -253,9 +253,9 @@ def _bulk_update_payments(opportunity: Opportunity, imported_data: Dataset) -> P
                 invalid_rows.append((row, "username required"))
             try:
                 amount = int(amount_raw)
+                payments[username] = amount
             except ValueError:
                 invalid_rows.append((row, "amount must be an integer"))
-            payments[username] = amount
```

Comment:
I think the "correct" way to do this is put it in an `else` clause. We don't want to catch errors with that statement (although I am not sure that is possible here), but only want it to run if the the `try` succeeds, and `else` is the standard way to do that.

Reply:
Thanks. Added. [14828cf](https://github.com/dimagi/commcare-connect/pull/480/commits/14828cfdb2c7ad103c12da26982781d1e8ce1c11)